from pathlib import Path
import numpy as np

path: Path
d1: np.lib.npyio.DataSource

<<<<<<< HEAD
d1.abspath(path)  # E: incompatible type
d1.abspath(b"...")  # E: incompatible type

d1.exists(path)  # E: incompatible type
d1.exists(b"...")  # E: incompatible type

d1.open(path, "r")  # E: incompatible type
d1.open(b"...", encoding="utf8")  # E: incompatible type
d1.open(None, newline="/n")  # E: incompatible type
=======
d1.abspath(path)  # type: ignore[arg-type]
d1.abspath(b"...")  # type: ignore[arg-type]

d1.exists(path)  # type: ignore[arg-type]
d1.exists(b"...")  # type: ignore[arg-type]

d1.open(path, "r")  # type: ignore[arg-type]
d1.open(b"...", encoding="utf8")  # type: ignore[arg-type]
d1.open(None, newline="/n")  # type: ignore[arg-type]
>>>>>>> 0a20124 (readme 수정)
