import numpy as np
import numpy.typing as npt

AR_i8: npt.NDArray[np.int64]
AR_f8: npt.NDArray[np.float64]

<<<<<<< HEAD
np.histogram_bin_edges(AR_i8, range=(0, 1, 2))  # E: incompatible type

np.histogram(AR_i8, range=(0, 1, 2))  # E: incompatible type

np.histogramdd(AR_i8, range=(0, 1))  # E: incompatible type
np.histogramdd(AR_i8, range=[(0, 1, 2)])  # E: incompatible type
=======
np.histogram_bin_edges(AR_i8, range=(0, 1, 2))  # type: ignore[arg-type]

np.histogram(AR_i8, range=(0, 1, 2))  # type: ignore[arg-type]

np.histogramdd(AR_i8, range=(0, 1))  # type: ignore[arg-type]
np.histogramdd(AR_i8, range=[(0, 1, 2)])  # type: ignore[list-item]
>>>>>>> 0a20124 (readme 수정)
