import numpy as np

<<<<<<< HEAD
np.testing.bob  # E: Module has no attribute
np.bob  # E: Module has no attribute

# Stdlib modules in the namespace by accident
np.warnings  # E: Module has no attribute
np.sys  # E: Module has no attribute
np.os  # E: Module "numpy" does not explicitly export
np.math  # E: Module has no attribute

# Public sub-modules that are not imported to their parent module by default;
# e.g. one must first execute `import numpy.lib.recfunctions`
np.lib.recfunctions  # E: Module has no attribute

np.__deprecated_attrs__  # E: Module has no attribute
np.__expired_functions__  # E: Module has no attribute
=======
np.testing.bob  # type: ignore[attr-defined]
np.bob  # type: ignore[attr-defined]

# Stdlib modules in the namespace by accident
np.warnings  # type: ignore[attr-defined]
np.sys  # type: ignore[attr-defined]
np.os  # type: ignore[attr-defined]
np.math  # type: ignore[attr-defined]

# Public sub-modules that are not imported to their parent module by default;
# e.g. one must first execute `import numpy.lib.recfunctions`
np.lib.recfunctions  # type: ignore[attr-defined]

np.__deprecated_attrs__  # type: ignore[attr-defined]
np.__expired_functions__  # type: ignore[attr-defined]
>>>>>>> 0a20124 (readme 수정)
