import numpy as np

<<<<<<< HEAD
class Test(np.nditer): ...  # E: Cannot inherit from final class

np.nditer([0, 1], flags=["test"])  # E: incompatible type
np.nditer([0, 1], op_flags=[["test"]])  # E: incompatible type
np.nditer([0, 1], itershape=(1.0,))  # E: incompatible type
np.nditer([0, 1], buffersize=1.0)  # E: incompatible type
=======
class Test(np.nditer): ...  # type: ignore[misc]

np.nditer([0, 1], flags=["test"])  # type: ignore[list-item]
np.nditer([0, 1], op_flags=[["test"]])  # type: ignore[list-item]
np.nditer([0, 1], itershape=(1.0,))  # type: ignore[arg-type]
np.nditer([0, 1], buffersize=1.0)  # type: ignore[arg-type]
>>>>>>> 0a20124 (readme 수정)
