from collections.abc import Sequence
from numpy._typing import _NestedSequence

a: Sequence[float]
b: list[complex]
c: tuple[str, ...]
d: int
e: str

<<<<<<< HEAD
def func(a: _NestedSequence[int]) -> None:
    ...

reveal_type(func(a))  # E: incompatible type
reveal_type(func(b))  # E: incompatible type
reveal_type(func(c))  # E: incompatible type
reveal_type(func(d))  # E: incompatible type
reveal_type(func(e))  # E: incompatible type
=======
def func(a: _NestedSequence[int]) -> None: ...

reveal_type(func(a))  # type: ignore[arg-type, misc]
reveal_type(func(b))  # type: ignore[arg-type, misc]
reveal_type(func(c))  # type: ignore[arg-type, misc]
reveal_type(func(d))  # type: ignore[arg-type, misc]
reveal_type(func(e))  # type: ignore[arg-type, misc]
>>>>>>> 0a20124 (readme 수정)
